data/clearance_astrazeneca/meta.yaml

```diff
@@ -0,0 +1,71 @@
+name: clearance_astrazeneca
+description: |-
+  Drug clearance is defined as the volume of plasma cleared of a drug
+  over a specified time period and it measures the rate at which the active drug
+  is removed from the body. This is a dataset curated from ChEMBL database containing
+  experimental results on intrinsic clearance, deposited from AstraZeneca. It
+  contains clearance measures from two experiments types, hepatocyte and microsomes.
+targets:
+- id: drug_clearance
+  description: The volume of plasma cleared of a drug over a specified time period
+    and it measures the rate at which the active drug is removed from the body
+  units: mL.min-1.g-1
+  type: continuous
+  names:
+  - Drug clearance
+  - The rate at which the active drug is removed from the body
+  - ADME Drug clearance
+  - Pharmacokinetics drug clearance
+  - Drug Excretion
+  - microsome/hepatocyte clearance
+  uris:
+  - https://bioportal.bioontology.org/ontologies/MESH?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMESH%2FD006207
```

Comment:
is this dataset actually looking at the half life?

Reply:
In a pharmacological manner yes

Reply:
I removed this from the `clearance_astrazeneca` dataset